.github/ISSUE_TEMPLATE/resolver-failure.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Dependency resolver failures / errors
+about: Report when the pip dependency resolver fails
+labels: ["K: UX", "K: crash", "C: new resolver", "C: dependency resolution"]
+
+---
+
+<!-- 
+  Please provide as much information as you can about your failure, so that we can understand the root cause.
+  
+  For example, if you are installing packages from pypi.org, we'd like to see:
+
+  - Your terminal output
+  - Any inputs to pip, for example:
+    - any package requirements: any CLI arguments and/or your requirements.txt file
+    - any already installed packages, outputted via `pip freeze`
+    
+  It would be great if you could also include your dependency tree. For this you can use pipdeptree: https://pypi.org/project/pipdeptree/
```

Comment:
FYI @uranusjr @pradyunsg - I added a line about pipdeptree based on a comment in Zulip from @uranusjr.
Please confirm if you're ok with this :)